Neos.Flow/Classes/Mvc/Controller/Argument.php

```diff
@@ -180,6 +181,11 @@ public function getDefaultValue()
     public function setValidator(ValidatorInterface $validator): Argument
     {
         $this->validator = $validator;
+        if ($validator !== null && $this->value !== null) {
```

Comment:
```suggestion
        // the validation should not be called on null values - for the cases where the value is required, the error will be thrown in Controller::mapRequestArgumentsToControllerArguments()
        if ($validator !== null && $this->value !== null) {
```
Does that sound clear enough or too elaborate @bwaidelich ? I'm pretty bad at condensing information...